rocAL/include/pipeline/commons.h

```diff
@@ -204,3 +204,12 @@ class BatchRNG {
     int64_t _seed;
     std::vector<RNG> _rngs;
 };
+
+/*! \brief MissingComponentsBehaviour for Webdataset
+ *
+ */
+enum MissingComponentsBehaviour {
```

Comment:
why do we have 2 enums for the same. We should use the one in api header

Reply:
One is exposed to the user and other is typecasted and used internally.